core/vm/state_manager.go

```diff
@@ -51,7 +51,6 @@ func callStateManager(input []byte, evm *EVM, contract *Contract) (ret []byte, e
 		return method(evm, contract, input)
 	}
 
-	ret, err = methodIds[methodID](evm, contract, input)
```

Comment:
Not sure how this ended up in here, but would result in a panic if the code path ever ended up here.